test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -309,7 +324,14 @@ def _undefined
   end
 
   # Set up grpc stubs to mock the external calls.
+  # TODO(qingling128): Remove this comment after grpc/12506 is resolved.
```

Comment:
Let's reference this properly, i.e., s/`grpc/12506`/`grpc/grpc#12506`/. Also below.

Reply:
Changed.